lib/compilers/dotnet.ts

```diff
@@ -160,36 +163,11 @@ class DotNetCompiler extends BaseCompiler {
         return this.compilerOptions;
     }
 
-    cleanAsm(stdout) {
-        let cleanedAsm = '';
-
-        for (const line of stdout) {
-            if (line.text.startsWith('; Assembly listing for method')) {
-                // ; Assembly listing for method ConsoleApplication.Program:Main(System.String[])
-                //                               ^ This character is the 31st character in this string.
-                // `substring` removes the first 30 characters from it and uses the rest as a label.
-                cleanedAsm = cleanedAsm.concat(line.text.substring(30) + ':\n');
-                continue;
-            }
-
-            if (line.text.startsWith('Emitting R2R PE file')) {
-                continue;
-            }
-
-            if (line.text.startsWith(';') && !line.text.startsWith('; Emitting')) {
-                continue;
-            }
-
-            cleanedAsm = cleanedAsm.concat(line.text + '\n');
-        }
-
-        return cleanedAsm;
-    }
-
     async runCrossgen2(compiler, execOptions, crossgen2Path, publishPath, dllPath, options, outputPath) {
         const crossgen2Options = [
             crossgen2Path, '-r', path.join(publishPath, '*'), dllPath, '-o', 'CompilerExplorer.r2r.dll',
             '--codegenopt', 'NgenDisasm=*', '--codegenopt', 'JitDiffableDasm=1', '--parallelism', '1',
+            '--inputbubble', '--compilebubblegenerics', '--resilient',
```

Comment:
please remove --resilient here, it will hide potential errors we don't want to see at all

Reply:
Done.